src/Corax/IndexSearcher/IndexSearcher.TermMatch.cs

```diff
@@ -1,9 +1,10 @@
 using System;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-using System.Text;
+using System.Text; //do not delete, required for DEBUG parts
```

Comment:
Instead of adding comment you could put the `using System.Text;` in between `#if DEBUG` directives. That should prevent automatic cleanup tools to remove it.